docs/source/supported_dependencies.rst

```diff
@@ -120,7 +120,10 @@ MAVIS also supports a :ref:`general VCF input <general-vcf-inputs>`. It should b
         - ``1.4.8 (custom)``
         - ``{indels/events_novel_exons,fusions/*}.tsv``
         - [Robertson-2010]_
-
+    *   - :term:`Strelka`
+        - ``1.0.6``
+        - ``passed.somatic.indels.vcf``
```

Comment:
Is the file suffix convention something that strelka always produces? or something that we just name the file normally? would someone else running strelka from scratch get a file with this naming convention? is this a suffix or the whole file name? if it is just a suffix you should add the glob asterisk in front

Reply:
I'm pretty sure "passed.somatic.indels.vcf" is a standard output file of strelka? I haven't run the strelka pipeline myself to know for sure.